config/application.config.php

```diff
@@ -2,7 +2,7 @@
 
 // Set up modules:
 $modules = [
-    'Zend\Router', 'ZfcRbac',
+    'Zend\Router', 'ZfcRbac', 'Zend\I18n', 'Zend\Mvc\I18n',
```

Comment:
This effectively adds the configuration provided by `Zend[\Mvc]\I18n\ConfigProvider`.